### PR TITLE
buffer() was removed in Python 3 so use bytes

### PR DESCRIPTION
**Pull request by cclauss**
https://docs.python.org/2.7/library/stdtypes.html#memoryview-type

https://portingguide.readthedocs.io/en/latest/etc.html#raw-buffer-protocol-buffer-and-memoryview

### CODE REVIEW

File: kombu/transport/qpid.py [hunk 1]
```diff
@@ -124,6 +124,11 @@
 
 logger = get_logger(__name__)
 
+try:
+    buffer
```
Comment (thedrow) [inline kombu/transport/qpid.py:128]: You should import buffer from `kombu.five` instead.

Comment (cclauss) [reply kombu/transport/qpid.py]: ImportError: cannot import name 'buffer' from 'vine.five'

**Comment by thedrow** [comment]
The tests are failing.
Please ensure that your refactor works.

**Comment by cclauss** [comment]
My refactor worked fine until https://github.com/celery/kombu/pull/1218#discussion_r453283999

Reverting to something that works.

**Pull request merged**